pyrobuf/parse_proto.py

```diff
@@ -41,6 +43,34 @@ class Parser(object):
         ('ENUM_FIELD', r'([A-Za-z_][0-9A-Za-z_]*)')
     ]
 
+    parsable_tokens = (
+        'OPTION',
+        'SYNTAX',
+        'IMPORT',
+        'MESSAGE',
+        'MODIFIER',
+        'FIELD',
+        'MAP_FIELD',
+        'LBRACKET',
+        'RBRACKET',
+        'DEFAULT',
+        'PACKED',
+        'DEPRECATED',
+        'COMMA',
+        'SEMICOLON',
+        'ENUM',
+        'LBRACE',
+        'RBRACE',
+        'EXTENSION',
+        'ONEOF',
+        'EXTEND',
+    )
+
+    tokens_not_supported_by_c = (
```

Comment:
Can we change this to just `unsupported_tokens`?

Reply:
`unsupported_tokens` is wiser, I just wanted to somehow communicate that it is supported by the parser, but not by the code that make it into a C extensions later on...Do you think there would be a good way to convey that?

Reply:
I take your input and put for some documentation if someone try to dive deeper